framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -336,8 +336,7 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
         this.visitorState = new VisitorState();
 
         this.loader = new AnnotationClassLoader(checker);
-        this.supportedQuals = createSupportedTypeQualifiers();
-        checkSupportedQuals();
+        this.supportedQuals = new HashSet<>();
```

Comment:
Jeff, this is not a raw type. It is better style to use diamonds and not give the explicit type arguments.